src/serializers/type_serializers/function.rs

```diff
@@ -488,6 +488,8 @@ struct SerializationInfo {
     include: Option<PyObject>,
     #[pyo3(get)]
     exclude: Option<PyObject>,
+    #[pyo3(get)]
+    context: Option<PyObject>,
```

Comment:
As `context` is arbitrary user contents which might end up creating a circular reference, it's probably a good idea to add a `__traverse__` implementation so that the Python GC can see into these fields. (Probably we should have already done this with `include` / `exclude`, though with proper use circular references seem less likely for those.)

Would you be willing to help add this please? 

https://pyo3.rs/v0.20.3/class/protocols#garbage-collector-integration

Reply:
[latest commit](https://github.com/pydantic/pydantic-core/pull/1215/commits/617d0864d54d8e8aa5f36c9630642017513f7765) should resolve this :)